MapboxNavigation/NavigationMapView.swift

```diff
@@ -126,6 +126,13 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     var altitude: CLLocationDistance
     var routes: [Route]?
     var isAnimatingToOverheadMode = false
+    var primaryRoutePoints: RoutePoints?
+    var primaryRouteLineGranularDistances: RouteLineGranularDistances?
+    var primaryRouteRemainingDistancesIndex: Int?
```

Comment:
Since NavigationMapView and vanishing route line is used only for one route let's rename these variables to:
```suggestion
    var routePoints: RoutePoints?
    var routeLineGranularDistances: RouteLineGranularDistances?
    var remainingDistancesIndex: Int?
```

Reply:
Thanks for the suggestion, I'll fix it